concepticondata/conceptlists/Scheible-2014-1755a.tsv

```diff
@@ -0,0 +1,1756 @@
+Scheible-2014-1755a-100	100	Buffet	buffet	noun			1743:Bedienung-Buffet:1 2291:Bedienung-Buffet:1 2891:Bedienung-Buffet:1	←67 ←67 ←67	ANT HYP SYN	5.5 0.5 1.8333
+Scheible-2014-1755a-101	101	Bundesland	federal state	noun			1819:Bundesland-Bundesrepublik:0 1823:Bundesland-Staat:0 2009:Bundesland-Region:0 2012:Bundesland-Verwaltungsbereich:0 2198:Bundesland-Gliedstaat:0 2201:Bundesland-Land:0 2348:Bundesland-Bundesrepublik:0 2349:Bundesland-Gliedstaat:0 2350:Bundesland-Land:0 2351:Bundesland-Region:0 2352:Bundesland-Staat:0 2353:Bundesland-Verwaltungsbereich:0 2967:Bundesland-Bundesrepublik:0 2971:Bundesland-Staat:0 3157:Bundesland-Region:0 3160:Bundesland-Verwaltungsbereich:0 3346:Bundesland-Gliedstaat:0 3349:Bundesland-Land:0	→102 →504 →451 →576 →244 →318 →102 →244 →318 →451 →504 →576 →102 →504 →451 →576 →244 →318	ANT ANT ANT ANT ANT ANT HYP HYP HYP HYP HYP HYP SYN SYN SYN SYN SYN SYN	3.3333 1.8333 1 1 0.3333 1.1667 5.8333 3.1667 6 6.1667 4.3333 4.8333 2.1667 2.3333 3.3333 3.8333 3.3333 2.6667
+Scheible-2014-1755a-102	102	Bundesrepublik	Federal Republic	noun			1819:Bundesland-Bundesrepublik:1 2348:Bundesland-Bundesrepublik:1 2967:Bundesland-Bundesrepublik:1	←101 ←101 ←101	ANT HYP SYN	3.3333 5.8333 2.1667
+Scheible-2014-1755a-103	103	Busch	bush	noun	1683	BUSH	2147:Pflanze-Busch:1 2621:Pflanze-Busch:1 3295:Pflanze-Busch:1	←420 ←420 ←420	ANT HYP SYN	0.8333 1.8333 4.6667
+Scheible-2014-1755a-104	104	Bushaltestelle	bus stop	noun			1723:Bahnsteig-Bushaltestelle:1 2275:Bahnsteig-Bushaltestelle:1 2871:Bahnsteig-Bushaltestelle:1	←57 ←57 ←57	ANT HYP SYN	3 0.3333 2.3333
+Scheible-2014-1755a-105	105	Bürger	citizens	noun	1032	CITIZEN	1776:Hoheit-Bürger:1 2457:Hoheit-Bürger:1 2924:Hoheit-Bürger:1	←265 ←265 ←265	ANT HYP SYN	4.5 2.5 0.5
+Scheible-2014-1755a-106	106	Bürgergesellschaft	civil society (members of the public)	noun			2160:Zivilgesellschaft-Bürgergesellschaft:1 2781:Zivilgesellschaft-Bürgergesellschaft:1 3308:Zivilgesellschaft-Bürgergesellschaft:1	←622 ←622 ←622	ANT HYP SYN	0.5 4 6
+Scheible-2014-1755a-107	107	Charakter	character	noun	1024	CHARACTER	2135:Persönlichkeit-Charakter:1 2610:Persönlichkeit-Charakter:1 3283:Persönlichkeit-Charakter:1	←418 ←418 ←418	ANT HYP SYN	0.3333 5 7.6667
```

Comment:
I would unmap because CHARACTER 1024 is defined as "A symbol used to represent a sound or a word."